kyo-core/shared/src/main/scala/kyo/vars.scala

```diff
@@ -54,3 +49,12 @@ object Vars:
     def run[V >: Nothing]: RunDsl[V] = new RunDsl[V]
 
 end Vars
+
+// Moving to Vars.internal crashes the compiler
```

Comment:
I get a cyclic type error for `Flat` if I do this, which doesn't make much sense. It seems a compiler bug.